.byebug_history

```diff
@@ -0,0 +1,43 @@
+q
```

Comment:
I recommend setting up your git installation to deal with these extra files via a global .gitignore file in your home dir: https://git-scm.com/docs/gitignore

Reply:
Thanks for bringing this to my attention, I just added a condition in the gitignore file to deal with this file type.